packages/thumbprint-react/components/StarRating/index.module.scss

```diff
@@ -1,77 +1,139 @@
-$spacing--small: 1px;
-$spacing--medium: 2px;
-$spacing--large: 3px;
+// Each image is a single gray star on top of a single yellow star.
 
-$size--small: 12px;
-$size--medium: 18px;
-$size--large: 24px;
+$small-spacing-adjustment: -1px;
+$small-height: 12px;
+$small-width: 70px;
+$small-image: url('data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTQiIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAxNCAyNCIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTcuMzQyMTUgMC4yNTc2MzFMOC42MjcyNSA0LjI0NTdMMTIuNjQyNCA0LjI5NDQzQzEyLjk4OCA0LjI5ODY2IDEzLjEzMSA0Ljc2MDE5IDEyLjg1MzkgNC45NzYxNEw5LjYzMjUgNy40ODkzNUwxMC44MjkgMTEuNTA3NEMxMC45MzE4IDExLjg1MzIgMTAuNTU3NyAxMi4xMzggMTAuMjc1NyAxMS45Mjg2TDcuMDAwMiA5LjQ5MzY2TDMuNzI0MyAxMS45MjgyQzMuNDQyNyAxMi4xMzc2IDMuMDY4MTMgMTEuODUyMiAzLjE3MDkzIDExLjUwN0w0LjM2NzUgNy40ODg4OEwxLjE0NjExIDQuOTc1NjdDMC44Njg5ODQgNC43NTk3MiAxLjAxMjAyIDQuMjk4MTkgMS4zNTc1MyA0LjI5Mzk2TDUuMzcyNzUgNC4yNDUyM0w2LjY1Nzg1IDAuMjU3MTYxQzYuNzY4NyAtMC4wODU3OTg2IDcuMjMxNzUgLTAuMDg1Nzk4NiA3LjM0MjE1IDAuMjU3NjMxWiIgZmlsbD0iI0QzRDRENSIvPgo8cGF0aCBkPSJNNy4zNDIxNSAxMi4yNTc2TDguNjI3MjUgMTYuMjQ1N0wxMi42NDI0IDE2LjI5NDRDMTIuOTg4IDE2LjI5ODcgMTMuMTMxIDE2Ljc2MDIgMTIuODUzOSAxNi45NzYxTDkuNjMyNSAxOS40ODk0TDEwLjgyOSAyMy41MDc0QzEwLjkzMTggMjMuODUzMiAxMC41NTc3IDI0LjEzOCAxMC4yNzU3IDIzLjkyODZMNy4wMDAyIDIxLjQ5MzdMMy43MjQzIDIzLjkyODJDMy40NDI3IDI0LjEzNzYgMy4wNjgxMyAyMy44NTIyIDMuMTcwOTMgMjMuNTA3TDQuMzY3NSAxOS40ODg5TDEuMTQ2MTEgMTYuOTc1N0MwLjg2ODk4NCAxNi43NTk3IDEuMDEyMDIgMTYuMjk4MiAxLjM1NzUzIDE2LjI5NEw1LjM3Mjc1IDE2LjI0NTJMNi42NTc4NSAxMi4yNTcyQzYuNzY4NyAxMS45MTQyIDcuMjMxNzUgMTEuOTE0MiA3LjM0MjE1IDEyLjI1NzZaIiBmaWxsPSIjRkVCRTE0Ii8+Cjwvc3ZnPgo=');
```

Comment:
@danoc It would be easier for maintenance (e.g., we want to update the star color / shape) if we're able to use an svg background image. I was getting `not found` errors on production build. Is there a way to do this?

Reply:
Hm, what do you mean by using an `svg` background image? Host it somewhere and link to the URL?

Reply:
Yes, include the svg assets in the component and reference by URL.

Reply:
Ah I see. 

The only way I can think of doing this is to host the image somewhere else and link to it here using the full URL. AFAIK the relative path won't work since the image wouldn't actually get uploaded.

In JavaScript, an imported image would get uploaded if the right Webpack loader is set up. But even that wouldn't help us because the SVG is used in `:after` and we can't get to that with an inline style.

If it works in IE, we could use this approach to improve maintainability a bit:
https://css-tricks.com/probably-dont-base64-svg/

Or leave a comment linking to the source file in Figma?

Reply:
Actually just realized we can convert back from base64 just as easily to get at the source SVG. So disregard this.